portal-impl/src/com/liferay/portal/upload/LiferayFileItem.java

```diff
@@ -137,6 +138,24 @@ public int getSizeThreshold() {
 	}
 
 	@Override
+	public File getStoreLocation() {
+		DeferredFileOutputStream dfos = null;
+
+		try {
+			dfos = (DeferredFileOutputStream)getOutputStream();
+		}
+		catch (IOException ioe) {
+			_log.error(ioe, ioe);
+		}
+
+		if (dfos == null) {
+			return null;
+		}
+
+		return dfos.getFile();
+	}
+
```

Comment:
Anyaway, since this bug only applies to WebLogic, i'm going to limit it scope to this server.

Are you ok with this??

Thanks!!!